### DISCUSSION TITLE
Pin mode for windows - how does it work?

### DISCUSSION THREAD

**Discussion by Aghyads** [rxhanson/Rectangle#1524] [Q&A]
Dear users, 
I'm new user to rectangle. Is it possible to Pin a window or app on the top of other oepened screens? like calculator, I want it to stay on top of excel even when clicking on excel app. Would that be possile with Rectangle pro? I have the pro version.
Thank you in advance for your response! 
Regards,

**Comment by Aaron-Rumpler** [comment]
This isn't something that macOS supports Rectangle doing. More details from @rxhanson here:
- [Feature | Stashed Windows Always on Top](https://github.com/rxhanson/RectanglePro-Community/issues/529)
- [Pin mode: Allow any application to float in front of all windows](https://github.com/rxhanson/Rectangle/discussions/1480)
- [Set a window to be always on top?](https://github.com/rxhanson/RectanglePro-Community/discussions/585)
